llama_stack/cli/download.py

```diff
@@ -48,6 +48,11 @@ def setup_download_parser(parser: argparse.ArgumentParser) -> None:
         required=False,
         help="See `llama model list` or `llama model list --show-all` for the list of available models",
```

Comment:
can we just make this `nargs='+'` so you can provide it multiple times like `--model-id A --model-id B`?